test/src/test/resources/logback-test.xml

```diff
@@ -44,8 +44,8 @@
     </logger>
 
     <!--<root level="DEBUG">-->
-        <!--&lt;!&ndash;<appender-ref ref="FILE" />&ndash;&gt;-->
```

Comment:
do you plan to revert this before merging?

Reply:
I think the old one is double '<!--'. The new one is cleaner.